api/v1/openapi.yaml

```diff
@@ -12,6 +12,23 @@ produces:
 consumes:
 - application/json
 paths:
+  "/cluster/nodes":
+    get:
+      summary: Get nodes information stored in the cilium-agent
+      tags:
+      - daemon
+      parameters:
+      - name: client-id
+        description: |
+          Client UUID that should be used to request new
```

Comment:
Is this description complete? It seems truncated. It also doesn't explain the difference between passing 0 or a non-zero UUID, nor does it explain the diff mechanism.